modules/monitoring/manifests/checks.pp

```diff
@@ -173,46 +173,70 @@
   # alerts will fire.
   $keep_last_value_limit = '132'
 
-  icinga::check::graphite { 'check_rummager_govuk_index_size_changed':
-    target              => "absolute(diffSeries(keepLastValue(stats.gauges.govuk.app.rummager.govuk_index.docs.count,${keep_last_value_limit}), timeShift(keepLastValue(stats.gauges.govuk.app.rummager.govuk_index.docs.count,${keep_last_value_limit}), \"7d\")))",
+  icinga::check::graphite { 'check_search_api_govuk_index_size_changed':
+    target              => "absolute(diffSeries(keepLastValue(stats.gauges.govuk.app.search-api.govuk_index.docs.count,${keep_last_value_limit}), timeShift(keepLastValue(stats.gauges.govuk.app.search-api.govuk_index.docs.count,${keep_last_value_limit}), \"7d\")))",
     warning             => 3000,
     critical            => 10000,
-    desc                => 'rummager govuk index size has significantly increased/decreased over the last 7 days',
+    desc                => 'search-api govuk index size has significantly increased/decreased over the last 7 days',
     host_name           => $::fqdn,
     notification_period => 'inoffice',
-    action_url          => "https://grafana.${app_domain}/dashboard/file/rummager_index_size.json",
-    notes_url           => monitoring_docs_url(rummager-index-size-change),
+    action_url          => "https://grafana.${app_domain}/dashboard/file/search_api_index_size.json",
+    notes_url           => monitoring_docs_url(search-api-index-size-change),
     from                => '25minutes',
     args                => "--dropfirst ${drop_first}",
   }
+  icinga::check::graphite { 'check_rummager_govuk_index_size_changed':
+    ensure    => 'absent',
+    target    => undef,
+    desc      => undef,
+    warning   => undef,
+    critical  => undef,
+    host_name => undef,
+  }
```

Comment:
Good practice :+1: but I don't think it's strictly necessary in this case, as the Icinga configuration directory is fully managed by Puppet, including purging files it doesn't recognise [1].

1: https://github.com/alphagov/govuk-puppet/blob/master/modules/icinga/manifests/config.pp#L43-L51